apps/web/src/components/providers/AuthProvider.tsx

```diff
@@ -60,14 +58,10 @@ export const useAuth = () => {
   return value;
 };
 
-export function getToken(): string {
-  const token = IS_EE_AUTH_ENABLED ? eeAuthTokenCookie.get() : getCommunityAuthToken();
-
-  return token || '';
-}
-
-export function getTokenClaims(): IJwtClaims | null {
-  const token = getToken();
+export async function getToken() {
+  if (IS_EE_AUTH_ENABLED) {
+    return (await window?.Clerk?.session?.getToken()) || '';
```

Comment:
Somehow, for me, in my PR `window. Clerk` was a little buggy with probably race-conditons. But when used without the `window` object, it was working fine.

Reply:
Weird, I guess it has to do with React renderings.